frontend/src/components/Footer.vue

```diff
@@ -0,0 +1,99 @@
+<script setup lang="ts"></script>
+
+<template>
+    <div id="footer">
+        <div class="container">
+            <div class="icons">
+                <a class="icon" href="https://fr-fr.facebook.com/BeninExcellence/">
+                    <i class="fa fa-facebook-f" />
+                </a>
+                <a class="icon" href="https://twitter.com/excellencebenin">
+                    <i class="fa fa-twitter" />
+                </a>
+                <a class="icon" href="#i">
+                    <i class="fa fa-instagram" />
+                </a>
+                <a class="icon" target="_blank" rel="noreferrer"
+                    href="https://www.linkedin.com/company/benin-excellence/">
+                    <i class="fa fa-linkedin" />
+                </a>
+            </div>
+            <ul class="copyright">
+                <li>&copy; Fondation Vallet</li>
+                <li>
+                    <a target="_blank" rel="noreferrer" href="https://www.fondationvallet.org/eeia/">
+                        École d'Été en Intelligence Artificielle (EEIA)
+                    </a>
+                </li>
+                <li>
+                    <a target="_blank" rel="noreferrer" href="#i">
```

Comment:
I couldn't find a good link for this one

Reply:
Maybe the LinkedIn profile again ?